test/smoke/src/sql/main.ts

```diff
@@ -34,12 +34,13 @@ export function main(isWeb: boolean = false): void {
 const PLATFORM = '${PLATFORM}';
 const RUNTIME = '${RUNTIME}';
 const VERSION = '${VERSION}';
+const ADS_SQLITE_RELEASE_VERSION = '1.5.0';
 
-const sqliteUrl = `https://github.com/Microsoft/azuredatastudio-sqlite/releases/download/1.5.0/azuredatastudio-sqlite-${PLATFORM}-${RUNTIME}-${VERSION}.zip`;
+const sqliteUrl = `https://github.com/Microsoft/azuredatastudio-sqlite/releases/download/${ADS_SQLITE_RELEASE_VERSION}/azuredatastudio-sqlite-${PLATFORM}-${RUNTIME}-${VERSION}.zip`;
 
 export async function setup(app: ApplicationOptions): Promise<void> {
 	console.log('*** Downloading test extensions');
-	const requestUrl = sqliteUrl.replace(PLATFORM, process.platform).replace(RUNTIME, getRuntime(app.web || app.remote || false)).replace(VERSION, getVersion(app.web || app.remote || false));
+	const requestUrl = sqliteUrl.replace(ADS_SQLITE_RELEASE_VERSION, getADSSqliteReleaseVersionForWeb(app.web || false)).replace(PLATFORM, process.platform).replace(RUNTIME, getRuntime(app.web || app.remote || false)).replace(VERSION, getVersion(app.web || app.remote || false));
```

Comment:
I think having a whole function for this isn't necessary. I'd simply just update the const to 
```
const sqliteUrl = `https://github.com/Microsoft/azuredatastudio-sqlite/releases/download/${RELEASE_VERSION}/azuredatastudio-sqlite-${PLATFORM}-${RUNTIME}-${VERSION}.zip`;
```


```
const releaseVersion = app.web ? '1.6.0' : '1.5.0';
const requestUrl = sqliteUrl.replace('${RELEASE_VERSION}', releaseVersion).replace(PLATFORM, process.platform).replace(RUNTIME, getRuntime(app.web || app.remote || false)).replace(VERSION, getVersion(app.web || app.remote || false));
```

That's much easier to follow and update.

Reply:
@Charles-Gagnon, where are environment variables like RELEASE_VERSION defined? Do those come from ADO or a file?

Reply:
These aren't environment variables. They're just static strings that are getting replaced.